docs/changelog/96161.yaml

```diff
@@ -4,3 +4,21 @@ area: "Search"
 type: enhancement
 issues:
  - 95541
+highlight:
+  title: Skip shards when querying constant keyword fields
+  body: "When a query like a match phrase query or a wildcard query targets a constant keyword field \
```

Comment:
I think wildcard query in 8.9.0 doesn't overwrite the `indexMetadataRewrite(...)` method? Only match_phrase and term queries.

Reply:
yes you are right...because it used the `SearchExecutionContext`.